packages/website/src/components/BigNumbers/StatsWithTitle.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { Stack, VStack } from '@coderscamp/ui/components/Stack';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { Statistic } from './Statistic';
+
+interface StatsWithTitleProps {
+  amounts: string[];
+  labels: string[];
+}
+
+export const StatsWithTitle = ({ amounts, labels }: StatsWithTitleProps) => {
+  const statisticStackDirection = useBreakpointValue({ base: 'column', xl: 'row' } as const);
+  const statisticFlexDirection = useBreakpointValue({ base: 'column', md: 'row' } as const);
+  const statisticSize = useBreakpointValue({ xl: 'min(1120px, 100%)', md: 'min(850px, 100%)' } as const);
```

Comment:
🙃 For consistency with previous two functions you can write `md` first and `xl` second in order